linux_os/guide/system/auditing/auditd_configure_rules/audit_kernel_module_loading/audit_rules_kernel_module_loading/rule.yml

```diff
@@ -56,18 +63,7 @@ references:
 
 ocil: |-
     {{{ ocil_audit_syscall(syscall="init_module") }}}
+    {{{ ocil_audit_syscall(syscall="finit_module") }}}
     {{{ ocil_audit_syscall(syscall="delete_module") }}}
```

Comment:
This needs to be broken into multiple rules, one per syscall (init_module, finit_module, delete_module).

Reply:
But this rule is aligning so nicely with item `Audit Kernel Module Loading and Unloading Events (Success/Failure) ` from https://github.com/ComplianceAsCode/content/pull/4680#issuecomment-521703084.